template-sample-app/template/src/components/views/login/Login.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Button, Container, Grid, makeStyles, Paper, Typography } from "@material-ui/core";
+import { useOAuthLogin } from "@carto/react/oauth";
+import { setTokenAndUserInfoAsync } from "@carto/react/redux";
+
+import { setError } from "config/appSlice";
+
+const useStyles = makeStyles((theme) => ({
+  containerWrapper: {
+    flex: "1 1 auto",
+    display: "flex",
+  },
+  paperForm: {
+    width: theme.spacing(40),
+    height: theme.spacing(32),
+    padding: theme.spacing(4)
+  }
+}));
+
+export default function Login() {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const oauthApp = useSelector((state) => state.oauth.oauthApp);
+
+  const onParamsRefreshed = (oauthParams) => {
+    if (oauthParams.error) {
+      dispatch(setError(`OAuth error: ${oauthParams.error}`));
+    } else {
+      dispatch(setTokenAndUserInfoAsync(oauthParams));
+    }
+  };
+
+  const [handleLogin] = useOAuthLogin(oauthApp, onParamsRefreshed);
+
+  return (
+    <Container className={classes.containerWrapper}>
```

Comment:
Maybe a Grid component here is better, unless you've found any issue to use it here

Reply:
Merging now, but I'll consider it when we apply the new design, thx!